scripts/lotus-daemon.service

```diff
@@ -1,19 +1,17 @@
 [Unit]
 Description=Lotus Daemon
-After=network-online.target
-Requires=network-online.target
+After=network.target
 
 [Service]
-Environment=GOLOG_FILE="/var/log/lotus/daemon.log"
-Environment=GOLOG_LOG_FMT="json"
+User=fc
+Group=fc
+ExecStartPre=/usr/local/bin/lotus-init.sh
 ExecStart=/usr/local/bin/lotus daemon
+ExecStop=/usr/local/bin/lotus daemon stop
+Environment=LOTUS_PATH=/var/lib/lotus
 Restart=always
-RestartSec=10
-
-MemoryAccounting=true
-MemoryHigh=8G
-MemoryMax=10G
-LimitNOFILE=8192:10240
+RestartSec=30
+TimeoutSec=infinity
```

Comment:
We definitely shouldn't set an infinite timeout sec

Reply:
It has to be pretty long to take into account the chain import. Maybe infinity isn't right, but I am not sure how long is reasonable before it becomes clear that an error has occurred, so instead I set this to infinity and rely on the start script to indicate if there's been an error.


I considered just not including long ExecStartPre at all, but I think do remove it could result in a poorer experience for people, particularly if when they install through packages. If you install an application, you expect it to be working correctly within a reasonable time-frame. Without importing the chain at the start, users are left either to wait for the chain to sync over the network or to find in the documentation that chain imports are possible and do it on their own. 

I think the way to do this might be to handle the import differently. Maybe --import-snapshot/--import-chain would only do do the import if it wasn't already done, or force to import again with --force-import-snapshot flags. 


Actually, I'll move this PR back into draft and come up with a better fix for this that doesn't require the intiialization script, because I think it does cause some problems, some of which are described in these comments.

Reply:
@coryschwartz Why not perform a oneshot service that does the import and have that be a prerequisite for starting the daemon service?

Reply:
Could even use a condition to determine if it needs to run.